app/src/debug/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt

```diff
@@ -31,16 +22,7 @@ import javax.inject.Inject
 class DebugMetricsHelper @Inject constructor() {
 
     internal fun init(context: Context) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/97f746770320b3a1fa5ca7613752fc49d440ea18/app/src/debug/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt#L24"></span>Function parameter context is unused.</td>
    </tr>
  </tbody>
</table>